config/puma.rb

```diff
@@ -11,7 +11,9 @@
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-port        ENV.fetch("PORT") { 80 }
+port=ENV.fetch("PORT") { 80 } 
+host=ENV.fetch("HOST") { "0.0.0.0" }
```

Comment:
Robocop complains about the missing spaces around `=` and the trailing spaces.

Reply:
Thanks for your reply, I've fixed that :) ( not used to automated checks )